setup/package.dfsrc

```diff
@@ -22,7 +22,6 @@ RUN apt-get update -qq &&  \
 		adduser \
 		libsass1 \
 		libxml2-dev \
-		libxslt1-dev \
```

Comment:
I'm not sure this is correct: the mexican localisation uses `etree.XSLT` which I would expect to rely on libxslt.

Reply:
It's a `-dev` package. Probably historically needed to compile lxml package.
I probably could remove all `-dev` packages.

Reply:
> I probably could remove all `-dev` packages.

Yes, at least those which underlie a package we get as binary (since there's still a `pip` call afterwards to fill things in I guess)

Reply:
I'll just try to remove them all and see if the docker image builds as expected. As the requirements are pip installed (line 42), thedocker build would crash if one of them is still needed.